### DISCUSSION TITLE
Problems when deep watching a fabric object in Vue.js

### DISCUSSION THREAD

**Discussion by MarcelBeks** [fabricjs/fabric.js#7107] [Q&A]
I am trying to deep watch a fabric js object for changes, but the problem I'm having is that is keeps firing constantly.
So my question is: Is there an attribute in the object or canvas that keeps changing over and over again, when i"m not moving the object or anything, that could cause this issue?

**Comment by asturur** [comment]
FabricJS objects are class instances.
I m not sure if deep watching a class instance is a good pattern. There could be some stuff that returns every time a new object or a new array.
I really do not know.
We have our own internal `deepWatch` thing, and i really dislike, i would like it to go away.
Can't you you just deepwatch a subset of properties? 
How are you deep watching it?
